packages/schemas/tables/logto_configs.sql

```diff
@@ -2,6 +2,6 @@ create table logto_configs (
   tenant_id varchar(21) not null
     references tenants (id) on update cascade on delete cascade,
   key varchar(256) not null,
-  value jsonb /* @use JsonObject */ not null default '{}'::jsonb,
+  value jsonb /* @use Json */ not null default '{}'::jsonb,
```

Comment:
By design ?

Reply:
By design, the value is not always a json object, and for the cases of `oidc.privateKeys` and `oidc.cookieKeys`, the value is an array of objects. Therefore, update the jsonb type to `Json`, which covers the array case.